students/WesleyM/Lesson3/mailroom.py

```diff
@@ -0,0 +1,47 @@
+donors = [
+  ["Alice", 20],
+  ["Bob", 100],
+  ["Charles", 30, 50, 10],
+  ["Denise", 10, 5],
+  ["Edward", 25, 20, 20, 20, 10],
+  ]
+
+def usein():
+  action = input("Choose one of three actions: \n 1. Send a Thank You \n 2. Create a Report \n 3. Quit \n>")
+  while (action.lower() not in ["1", "2", "3", "send a thank you", "create a report", "quit"]):
```

Comment:
You don't need parentheses around the `while` clause and you might want to assign this list to a variable outside the while block for readability so you could say something like `while action.lower() not in options`